sdk/test-utils/recorder/src/utils/index.ts

```diff
@@ -436,8 +436,39 @@ export function handleSingleQuotesInUrlPath(fixture: string): string {
   return updatedFixture;
 }
 
+/**
+ * Meant for node recordings only!
+ *
+ * Masks access tokens in the json response from nock fixtures.
+ * For example, the following part of the nock fixture/recording would be updated.
+ * from `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"e6z-9_g"}, [`
+ * to   `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"access_token"}, [`
+ *
+ * @param {string} fixture
+ */
+export function maskAccessTokenInNockFixture(fixture: string): string {
+  // Replaces only if the content-type is json
+  if (!isBrowser() && isContentTypeInNockFixture(fixture, jsonContentTypes)) {
+    // Matches the nock's reply from the fixture
+    const matches = fixture.match(/\.reply\((.*), (.*), .*/);
+    if (matches && matches[2]) {
+      // TODO: Only replace if the "token_type" is "Bearer"??
```

Comment:
AAD only supports bearer tokens, so you should never get a `token_type` other than Bearer.

Reply:
Thanks for the info!

Reply:
Removed the TODO